src/posts/2020-01-27-stencil-getting-started.md

```diff
@@ -0,0 +1,70 @@
+---
+title: 'Aan de slag met Stencil'
+tags: Stencil, Web Components, Design System
+comments: true
+author: danny
+min_read: 4
+date: 2020-01-27
+---
+
+### Wat is Stencil?
+[Stencil](https://stenciljs.com/) is een compiler die Web Components (Custom Elements) genereert. Stencil combineert de beste concepten van de meest populaire frameworks in een simpele tool.
+
+Stencil maakt gebruik van features zoals:
+* Virtual DOM
+* Async rendering
+* Reactive data-binding
+* TypeScript
+* JSX
+
+Stencil is ontwikkeld dor het Ionic framework team.
```

Comment:
```suggestion
Stencil is ontwikkeld door het Ionic framework team.
```

Reply:
Thanks for the heads up 😄, I overlooked that typo, fixed it in bf67141.